samples/OneClickDemo/OneClickDemo-OMD.sh

```diff
@@ -25,6 +27,7 @@ if [[ "$FYBRIK_VERSION" == "" ]]; then
   # Get Fybrik lateset realease from github
   FYBRIK_VERSION=$(git -c 'versionsort.suffix=-' ls-remote --tags --sort='v:refname' https://github.com/fybrik/fybrik.git | tail --lines=1 | cut --delimiter='/' --fields=3)
 fi
+FYBRIK_BRANCH_OMD="$FYBRIK_VERSION"
```

Comment:
why do you need a separate `FYBRIK_BRANCH_OMD` env variable?

Reply:
it is as it is installed in the [QuickStart](https://fybrik.io/v1.3/get-started/quickstart/#install-data-catalog), it is used for the [OpenMetaData script](https://github.com/fybrik/fybrik/blob/master/third_party/openmetadata/install_OM.sh#L9), but it needs to be FYBRIK_BRANCH and exported for it to work for that.
I changed it accordingly.